pkg/wing/server/server.go

```diff
@@ -91,28 +88,10 @@ func (o WingServerOptions) Config() (*apiserver.Config, error) {
 	if err := o.RecommendedOptions.SecureServing.MaybeDefaultWithSelfSignedCerts("localhost", nil, []net.IP{net.ParseIP("127.0.0.1")}); err != nil {
 		return nil, fmt.Errorf("error creating self-signed certificates: %v", err)
 	}
-
-	serverConfig := genericapiserver.NewConfig(apiserver.Codecs)
-	if err := o.RecommendedOptions.Etcd.ApplyTo(serverConfig); err != nil {
-		return nil, err
-	}
-	if err := o.RecommendedOptions.SecureServing.ApplyTo(serverConfig); err != nil {
-		return nil, err
-	}
-
-	client, err := clientset.NewForConfig(serverConfig.LoopbackClientConfig)
-	if err != nil {
+	serverConfig := genericapiserver.NewRecommendedConfig(apiserver.Codecs)
+	if err := o.RecommendedOptions.ApplyTo(serverConfig); err != nil {
 		return nil, err
 	}
-	informerFactory := informers.NewSharedInformerFactory(client, serverConfig.LoopbackClientConfig.Timeout)
-	admissionInitializer, err := winginitializer.New(informerFactory)
-	if err != nil {
-		return nil, err
-	}
-	if err := o.Admission.ApplyTo(serverConfig, admissionInitializer); err != nil {
-		return nil, err
-	}
-
```

Comment:
Why have you removed this? This is required for admission controllers in wing. Pinning to release branches shouldn't require deleting much code, just changing some types around.